File: ITManagementServer/src/main/java/models/responseModels/departments/DepartmentGridResponseModel.java
package models.responseModels.departments;

public class DepartmentGridResponseModel {
    private int DepartmentId;
    private String Title;
    private String WorkerDuties;
    private int WorkersCount;

    public int getDepartmentId() {
        return DepartmentId;
    }

    public void setDepartmentId(int departmentId) {
        DepartmentId = departmentId;
    }

    public String getTitle() {
        return Title;
    }

    public void setTitle(String title) {
        Title = title;
    }

    public String getWorkerDuties() {
        return WorkerDuties;
    }

    public void setWorkerDuties(String workerDuties) {
        WorkerDuties = workerDuties;
    }

    public int getWorkersCount() {
        return WorkersCount;
    }

    public void setWorkersCount(int workersCount) {
        WorkersCount = workersCount;
    }
}

File: ITManagementClient/Models/ResponseModels/Departments/DepartmentGridResponseModel.cs
namespace ITManagementClient.Models.ResponseModels.Departments
{
    public class DepartmentGridResponseModel
    {
        public int DepartmentId { get; set; }

        public string Title { get; set; }

        public string WorkerDuties { get; set; }

        public int WorkersCount { get; set; }
    }
}